devolutions-gateway/src/plugin_manager/registry.rs

```diff
@@ -0,0 +1,334 @@
+use crate::{
```

Comment:
I feel this file does not belong to this module because this isn't a plugin. Same goes for `push_files.rs`.

Reply:
removed it to the registry directory. The registry file is now called mod.rs

Reply:
Great!